### DISCUSSION TITLE
Can you use the integration without the IKEA app?

### DISCUSSION THREAD

**Discussion by Lotok** [sanjoyg/dirigera_platform#120] [Q&A]
Hello
I'm new to smart homes, so I may not understand this correctly. I thought that with this integration, I could link my hub and start adding my components, like bulbs, without using the IKEA app. Is this the correct intent of this integration, or am I misunderstanding it? I've added my Dirigera hub to Home Assistant, but I can't add my bulb.
Thank you for your patience with me!

**Comment by sanjoyg** [comment]
No that is not correct. You have to add the components to the IKEA Hub
